anax-in-k8s/Dockerfile

```diff
@@ -20,8 +20,10 @@ WORKDIR /home/anaxuser
 COPY script/* /home/anaxuser/
 COPY config/* /etc/horizon/
 
-ADD bin/* /usr/horizon/bin/ 
-ADD bin/* /usr/bin/
+ADD anax /usr/horizon/bin/
+ADD hzn /usr/horizon/bin/ 
+ADD anax /usr/bin/
```

Comment:
What is the reason for 2 copies of the binaries?

Reply:
anax.service will need them from /usr/horizon/bin/, and hzn register will need them from /usr/bin/
I will try add them to /usr/horizon/bin/ and update PATH

Reply:
Oh, maybe I can just add anax to /usr/horizon/bin/ and add hzn to /usr/bin/